applications/PfemFluidDynamicsApplication/custom_processes/remove_mesh_nodes_for_fluids_process.hpp

```diff
@@ -407,13 +403,11 @@ namespace Kratos
 				// coordinates
 				for (unsigned int i = 0; i < ie->GetGeometry().size(); i++)
 				{
-
 					if ((ie->GetGeometry()[i].Is(RIGID) && ie->GetGeometry()[i].IsNot(INLET)) || ie->GetGeometry()[i].Is(SOLID))
```

Comment:
you can declar a `const auto& r_geometry = GetGeometry()` and the reuse the `r_geometry`